packages/nest/src/generators/convert-tslint-to-eslint/schema.json

```diff
@@ -21,6 +21,11 @@
       },
       "x-prompt": "Which NestJS project would you like to convert from TSLint to ESLint?"
     },
+    "discardExistingLintConfig": {
```

Comment:
Can we also save this option in `workspace.json` so they don't have to answer the question anymore?

Reply:
Done